docs/docs/adding-redux-store.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Adding a Redux Store"
+---
+
+[Redux](https://redux.js.org/) helps you write applications that behave consistently, run in different environments (client, server, and native), and are easy to test; that’s why Gatsby uses it as a core technology under the hood. On top of that, Redux provides a great developer experience, such as [live code editing combined with a time-traveling debugger](https://github.com/reduxjs/redux-devtools).
+
+In order to use Redux for custom state management in a Gatsby site, you'll need to hook into two of Gatsby's extension points:
```

Comment:
```suggestion
In order to use Redux for **custom** state management in a Gatsby site, you'll need to hook into two of Gatsby's extension points:
```
Emphasizing "custom" here might be a good idea so people don't get the wrong impression that this is required.